requirements.txt

```diff
@@ -0,0 +1,2 @@
+icecream==2.1.3
+pytest==8.2.1
```

Comment:
zawsze miej plik który ci śledzi zależności, z konkretnymi wersjami paczek. Zaoszczędzi to całe mnogie kłopotów jeżeli ktoś inny będzie chciał przy tym pracować (albo ty będziesz chciał na nowym komputerze czy coś). Są różne sposoby i podejścia, ale się sprowadzają do czegoś takiego.
`requirements.txt` jest standardowym plikiem w Pythonie.

Reply:
instaluje się z takiego pliku przez `pip install -r requirements.txt`, ale w tym przypadku `pip install -e .` to zrobi automagicznie

Reply:
Fajne :)